Quantum-Control-Applications/Superconducting/Single Fixed Transmon/configuration.py

```diff
@@ -136,7 +138,8 @@ def readout_macro(threshold=None, state=None, I=None, Q=None):
 minus_y90_wf, minus_y90_der_wf = np.array(
     drag_gaussian_pulse_waveforms(minus_y90_amp, minus_y90_len, minus_y90_sigma, alpha=drag_coef, delta=anharmonicity)
 )
-minus_y90_der_wf = (-1) * minus_y90_der_wf
+minus_y90_I_wf = (-1) * minus_y90_der_wf
+minus_y90_Q_wf = minus_y90_wf
```

Comment:
Shall we use this formalism (I & Q instead of _wf & der_wf) in the other scripts in qua_libs as well

Reply:
I agree

Reply:
We can also add it for the `x` pulses.
`x180_I_wf = x180_wf`
`x180_Q_wf = x180_der_wf`

Reply:
yes, that is a good idea, doing this.